rfcs/consume_user_activation.md

```diff
@@ -0,0 +1,41 @@
+# RFC 128: Consume user activation 
+
+Editor: Marcos Caceres, Apple Inc.
+
+## Summary 
+
+Tests in WPT are currently relying on various indirect means to "[consume user activation](https://html.spec.whatwg.org/#consume-user-activation)" [HTML].
+Some of these means are non-standard in as far as they are not specified to consume user activation, meaning that the tests are inshrining non-standard behavior for conformance purposes.
+
+The workarounds being used are as follows, with several drawbacks:
+
+* Using Fullscreen API: non-standard, takes significant time to put an element into fullscreen. 
```

Comment:
```suggestion
* Using Fullscreen API: it takes significant time to put an element into fullscreen. 
```

Reply:
As https://github.com/whatwg/fullscreen/pull/153 is about to land, let’s treat it as standard.